Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2367 @@
+    if page_result:
+        readable_output, context_data = incident_event_readable_output(page_result, title)
+    else:
+        readable_output = 'No Event for Incidents data to present.'
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix=f'{INTEGRATION_CONTEXT_NAME}.IncidentEvent',
+        outputs_key_field='event_uuid',
+        outputs=context_data,
+        raw_response=raw_response,
+        ignore_auto_extract=True
+    )
+
+
+def get_incident_list_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """
+    Get Incident List
+    Args:
+        client: Symantec EDR on-premise client objectd to use.
+        args: all command arguments, usually passed from ``demisto.args()``.
+    Returns:
+        CommandResults: A ``CommandResults`` object that is then passed to ``return_results``, that contains an updated
+            result.
+    """
+    context_data: list = []
+    endpoint = '/atpapi/v2/incidents'
+
+    payload, limit, offset, page_size = get_request_payload(args, 'incident')
+    demisto.debug(f'Incident Payload: {payload}')
+    raw_response = client.query_request_api(method='POST', url_suffix=endpoint, params={}, json_data=payload)
+    title = compile_command_title_string(
+        'Incident',
+        arg_to_number(args.get('page', 0)),
+        page_size,
+        arg_to_number(raw_response.get('total'))
+    )
+
+    page_result = get_data_of_current_page(offset, limit, raw_response.get('result', []))
+    if page_result:
+        readable_output, context_data = incident_readable_output(page_result, title)
+    else:
+        readable_output = 'No Incidents data to present.'
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix=f'{INTEGRATION_CONTEXT_NAME}.Incident',
+        outputs_key_field='apt_incident_id',
+        outputs=context_data,
+        raw_response=raw_response
+    )
+
+
+def get_incident_comments_command(client: Client, args: dict[str, Any]) -> CommandResults:
+    """
+    Get all comments based on Incident ID
+    Args:
+        client: Symantec EDR on-premise client objectd to use.
+        args: all command arguments, usually passed from ``demisto.args()``.
+    Returns:
+        CommandResults: A ``CommandResults`` object that is then passed to ``return_results``, that contains an updated
+            result.
+    """
+    context_data: list = []
+
+    # Get UUID based on incident_id
+    uuid = get_incident_uuid(client, args)
+    endpoint = f'/atpapi/v2/incidents/{uuid}/comments'
+
+    # Argument incident_id not be required in further incident command query therefor
+    # remove it from args
+    incident_id = args.pop("incident_id", None)
+    payload, limit, offset, page_size = get_request_payload(args, 'incident')
+    raw_response = client.query_request_api(method='POST', url_suffix=endpoint, params={}, json_data=payload)
```

Comment:
instead of defining the `endpoint` and using `query_request_api`, create a Client method called `get_incident_comments` that has the endpoint hardcoded and performs the request. 

the `query_request_api` method should be replaced with calls to `Client.http_request` inside the new `get_incident_comments` command (and all other calls to `query_request_api` should be replaced as well)